src/cpu-template-helper/src/template/dump/aarch64.rs

```diff
@@ -34,9 +37,24 @@ pub fn config_to_template(cpu_config: &CpuConfiguration) -> CustomCpuTemplate {
     }
 }
 
+// List of register IDs excluded from the CPU configuration dump.
+static REG_EXCLUSION_LIST: &[u64] = &[
+    // SYS_CNTV_CVAL_EL0 and SYS_CNTPCT_EL0 are timer registers and depend on the elapsed time.
+    // This type of registers are not useful as guest CPU config dump.
+    SYS_CNTV_CVAL_EL0,
+    SYS_CNTPCT_EL0,
+    // Program counter (PC) value is determined by the given kernel image. It should not be
+    // overwritten by a custom CPU template and does not need to be tracked in a fingerprint file.
+    PC,
+];
+
+fn should_exclude_reg(reg_id: u64) -> bool {
+    REG_EXCLUSION_LIST.contains(&reg_id)
+}
```

Comment:
This function is only called in 1 place and is 1 line, it is more readable to have this 1 line in this one place (there is lint along these lines https://rust-lang.github.io/rust-clippy/master/index.html#/single_call_fn).

Reply:
fixed